Resources/Prototypes/_Harmony/Entities/Mobs/NPCs/animals.yml

```
@@ -0,0 +1,15 @@
+﻿- type: entity
+  name: dividing goose
+  parent: MobGoose
+  id: MobDividingGoose
+  components:
+  - type: ActionGrant
+    actions:
+      - DividingGoose
```

Lists should not have an indent
```suggestion
    actions:
    - DividingGoose
```